### ISSUE TITLE
ERROR: Chromosome records in alignment file and VCF are sorted differently.

### ISSUE THREAD

**Issue by msozzoni** [whatshap/whatshap#303]
Hello,
I used the command:
`whatshap haplotag -o file.bam --reference reference.fasta phased.vcf.gz file.bam`
and I got this error
ERROR: Chromosome records in alignment file and VCF are sorted differently.

I've seen that there are issues still open with the same problem, did you found a solution to this problem?

Thanks a lot

**Comment by marcelm** [comment]
Yes, this appears to be the same as issue #140. Let’s continue the discussion over there.

**Comment by marcelm** [comment]
Hm, I can actually not reproduce the problem with the test data given in #140, so perhaps it’s a different one.

Which WhatsHap version are you using?

**Comment by msozzoni** [comment]
I'm using the 0.14.1

**Comment by marcelm** [comment]
Please update to a more recent version. As far as I can tell, the problem has been solved in the meantime.
